api/src/main/java/org/grammatek/infrastructure/ApiClient.kt

```diff
@@ -40,7 +44,7 @@ open class ApiClient(val baseUrl: String) {
         var username: String? = null
         var password: String? = null
         var accessToken: String? = null
-        const val baseUrlKey = "org.openapitools.client.baseUrl"
+        const val baseUrlKey = "org.grammatek.baseUrl"
```

Comment:
Where is this value defined? Should this be the default url in case there is none provided ? Then we should change this to `yfirlestur.grammatek.com`

Reply:
I believe this is just a key and it should match the package name. 
You can see it being used in the UsersApi.kt 
           `System.getProperties().getProperty(key = ApiClient.baseUrlKey, defaultValue = "https://yfirlestur.is")`

Reply:
ok